spec/src/main/asciidoc/fault-tolerance-interceptor.asciidoc

```diff
@@ -33,25 +33,32 @@ For instance, in the following example, `MyLogInterceptor` will be invoked first
 [source, java]
 ----
 @Retry
-@MyLog 
+@MyLog
 @MyPrint
-public void myInvok() {
-
- //do something
+public void myInvoke() {
+  // do something
 }
+
 @Priority(3000)
 @MyLog
-public class MyLogInterceptor{
- //do logging
+public class MyLogInterceptor {
+  // do logging
 }
 
 @Priority(5000)
 @MyPrint
-public class MyPrintInterceptor{
- //do printing
+public class MyPrintInterceptor {
+  // do printing
 }
 ----
 
+=== Treating custom throwables
```

Comment:
I think either "Treatment of custom throwables" or just "Custom throwables" would work better as the heading on this section.

Reply:
Good point, changed to "Custom throwables".